cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,14 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io/#login');
 
+    cy.get('input[type="email"]').type('ruviw@mailinator.com');
+
+    cy.get('input[type="password"]').type('Pa$$w0rd!');
+
+    cy.get('.btn').click();
+
+    cy.get('nav').should('contain', 'Eaque65575');
```

Comment:
The username 'Eaque65575' seems to be hardcoded. If this value is expected to change, it should not be hardcoded.